modules/gdscript/gdscript_editor.cpp

```diff
@@ -403,6 +403,11 @@ void GDScriptLanguage::debug_get_globals(List<String> *p_globals, List<Variant>
 
 		const Variant &var = gl_array[E.value];
 		if (Object *obj = var) {
+			bool freed = false;
+			var.get_validated_object_with_check(freed);	
```

Comment:
Maybe we can create another method in Variant to just check the validity?

Reply:
Not needed as you'd always just be fetching the object, and not needed for this case see below